modelos/Endereco.js

```diff
@@ -6,4 +6,10 @@ class endereco {
         this.bairro = bairro
         this.cidade = cidade
     }
+    getRua (){
+        return this.rua
+    }
+    setRua (){
```

Comment:
Esse metodo é um set
serRua(rua) {
    this.rua = rua;
}

Reply:
Corrigi verifica por favor

Reply:
Tudo certo, aprovado.